lib/scale/enc_dec_test.go

```diff
@@ -168,3 +168,24 @@ func TestEncodeDecodeCustom_InsideStruct(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, tt, dec)
 }
+
+type myBytes [64]byte
+
+func TestEncodeDecodeCustom_Array(t *testing.T) {
+	b := myBytes([64]byte{1, 2, 3, 4})
+	enc, err := Encode(b)
+	require.NoError(t, err)
+
+	_, _ = Decode(enc, new(myBytes))
```

Comment:
It could be useful to add a require.Nil on err instead of ignoring it. Or I'm missing something?

Reply:
this isn't supported, it is supposed to error. the test is making sure it doesn't panic as per the linked issue #947